elasticmagic/expression.py

```diff
@@ -454,6 +454,9 @@ def missing(self, **kwargs):
     def in_(self, terms, **kwargs):
         return Terms(self, terms, **kwargs)
 
+    def not_in_(self, terms, **kwargs):
```

Comment:
Sqlalchemy has `notin_` method so I think we also should use the same name for consistency.